queries.sql

```diff
@@ -1,3 +1,10 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM animals WHERE name LIKE '%mon';
+SELECT * FROM animals WHERE EXTRACT(YEAR FROM date_of_birth) BETWEEN 2016 AND 2019;
+SELECT * FROM animals WHERE neutered = TRUE AND escape_attempts < 3;
+SELECT * FROM animals WHERE name = 'Agumon' OR name = 'Pikachu';
+SELECT * FROM animals WHERE weight_kg > 10.5;
```

Comment:
- [ ] Nice attempt here. However, the requirement was for `name, escape attempts` of animals that weigh `10.5kg`. Kindly refactor your query to capture the missing two requirements 👍 

## Requirements:
![image](https://user-images.githubusercontent.com/50990925/235603395-4571b42a-ac4b-4cd7-b228-c69c18b7c956.png)